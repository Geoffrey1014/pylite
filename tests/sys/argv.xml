lambda
  name __init__
  param_count 0
  local_count 1
  label_count 3
  insn_vector
    load_module c0
    load_this_module
    store_field c0
    load_global c0
    load_field c1
    load_global c2
    call 1
    print
    load_global c0
    load_field c1
    iterator
    store_local -1
    jmp L1
    label L0
    store_global c3
    load_global c3
    print
    label L1
    load_local -1
    next L0
    pop
    label L2
    load_const c4
    return
  const_vector
    sys
    argv
    len
    arg
    None
  guard_vector
  line_number_map 1 1 1 3 3 3 3 3 4 4 4 4 4 4 4 5 5 4 4 4 4 4 1 1
