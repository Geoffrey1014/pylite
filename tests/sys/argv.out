1
./sys/argv.py
